tests/MongoDB.Driver.Core.Tests/Core/Connections/BinaryConnection_CommandEventTests.cs

```diff
@@ -47,6 +48,16 @@ public class BinaryConnection_CommandEventTests : IDisposable
         private BinaryConnection _subject;
         private IDisposable _operationIdDisposer;
 
+        public static IEnumerable<object[]> GetPotentiallyRedactedCommands()
+        {
+            var potentiallyRedactedCommands =
+                CommandEventHelperReflector.__securitySensitiveCommands.ToList();
+            var conditionallyRedactedCommands =
+                CommandEventHelperReflector.__conditionallySecuritySensitiveCommands.Keys.ToList();
+            potentiallyRedactedCommands.AddRange(conditionallyRedactedCommands);
+            return potentiallyRedactedCommands.Select(c => new object[] { c });
+        }
+
```

Comment:
Let's name the method more appropriately and go low tech:

    public static IEnumerable<object[]> GetPotentiallyRedactedCommandTestCases()
    {
        var potentiallyRedactedCommands = new[]
            {
                "authenticate",
                "saslStart",
                "saslContinue",
                "getnonce",
                "createUser",
                "updateUser",
                "copydbgetnonce",
                "copydbsaslstart",
                "copydb",
                "isMaster",
            };
        return potentiallyRedactedCommands.Select(c => new object[] { c });
    }

Reply:
Done